twisted/conch/ssh/keys.py

```diff
@@ -39,6 +42,47 @@
 from twisted.python.deprecate import deprecated, getDeprecationWarningString
 from twisted.python.versions import Version
 
+# Curve lookup table
+curveTable = {
```

Comment:
If they are going to remain (whether public or private) each should have a `@var` declaration in the module docstring.